rsopt/configuration/options/__init__.py

```diff
@@ -69,6 +68,21 @@ def _check_options(cls, options):
         for key in cls.REQUIRED_OPTIONS:
             assert options.get(key), f"{key} must be defined in options for {name}"
 
+    @property
+    def objective_function(self) -> typing.List[str]:
+        _obj_f = self._objective_function.copy()
+        if len(self._objective_function) == 2:
+            _obj_f[0] = str(pathlib.Path(self._objective_function[0]).resolve())
```

Comment:
Might want to work on the copy, _obj_f, just to keep things a bit more atomic. This holds true for the checks elsewhere in here as well